### DISCUSSION TITLE
Restore exclusions?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [rustic-rs/rustic#516] [Q&A] [answered]
Is not possible to exclude paths from restoring? I wanted to restore a particular snapshot but wanted to exclude some paths, in restic it seems we could use --exclude, is there something similar in rustic?

**Comment by aawsome** [comment]
Currently, there is only the possibility to restrict your restore to a sub-path using the `restore <snap>:/my/path` syntax. Please open an issue if you need such an option!

**Comment by aawsome** [comment, marked as answer]
Exclude options for restore has been added in #572.
